ckcp/gitops.sh

```diff
@@ -8,11 +8,11 @@ SCRIPT_DIR="$(cd "$(dirname "$0")" >/dev/null ; pwd)"
 GITOPS_DIR="$(dirname "$SCRIPT_DIR")/gitops"
 WORK_DIR="$SCRIPT_DIR/work"
 KUBECONFIG_CLUSTER="${KUBECONFIG:-$HOME/.kube/config}"
```

Comment:
if KUBECONFIG_CLUSTER is not used anywhere, can we remove it? 
(its only used in line 13 and that can be replaced with KUBECONFIG directly)

lines 10/13
KUBECONFIG="${KUBECONFIG:-$HOME/.kube/config}"
KUBECONFIG_MERGED="merged-config.kubeconfig:$KUBECONFIG:$KUBECONFIG_KCP"

Reply:
updated